### DISCUSSION TITLE
Error with metadata

### DISCUSSION THREAD

**Discussion by dstepanian** [deepjyoti30/ytmdl#232] [Q&A]
[metadata]:[20/04/2022 17:44:51]:  Something went wrong with iTunes. The program will continue with the other providers. Please check '[C:\Users\**********\.cache\ytmdl\logs\log.cat]' for more details.

**Comment by deepjyoti30** [comment]
@dstepanian Is this happening for every song or just for a specific one?

**Comment by poisonedsandwich** [comment]
Im getting the same problem. i'll just leave the log, maybe that'll help
[log.txt](https://github.com/user-attachments/files/18844685/log.txt)
